documentation/airport-guides/egll.md

```diff
@@ -0,0 +1,67 @@
+### About London Heathrow
+
+Heathrow Airport (Also known as London Heathrow) (ICAO: LKPR, IATA: LHR) is the main and biggest airport in the United Kingdom, acting as the primary hub for British Airways (ICAO: BAW, callsign: SPEEDBIRD), and the primary operating airport for Virgin Atlantic (ICAO: VIR, callsign: VIRGIN). 
+
+Heathrow Airport serves the majority of London air travel, as well as the majority of southern Britain. It is located 14 miles to the West of London's City Center. It is one of the six london airports servicing London (Along with City, Stansted, Luton, Southend and Gatwick). 
+
+Heathrow airport started in 1929 as a small airfield in the hamlet of Heathrow. The development of Heathrow picked up pace in 1944 as a stopover location for long-distance military aircraft, however, by the time it neared completion WW2 had already finished, and the British government continued to develop the airport for civil use. It was opened on 25 March 1946 named 'London Airport', and renamed to 'Heathrow Airport' in 1966. 
+
+### Statistics
+
+78,000,000 passengers in 2017
+474,033 aircraft movement in 2017
+
+[Source](https://www.heathrow.com/company/company-news-and-information/company-information/facts-and-figures)
+
+### Airspace
+
+London has a polygonal airspace serving all six London's airports. More info on the London TMA can be found in [this NATS chart](http://www.ead.eurocontrol.int/eadbasic/pamslight-E89151B3C539313B4E86AD7E95B6927A/7FE5QZZF3FXUS/EN/Charts/ENR/AIRAC/EG_ENR_6_2_1_11_en_2018-03-29.pdf). 
```

Comment:
Change to:
`The London TMA, being responsible for all of London's airports, covers an enormous area from Southampton to Chelmsford. For more information, check the [NATS website](http://www.nats-uk.ead-it.com/public/index.php%3Foption=com_content&task=blogcategory&id=165&Itemid=3.html)`
(The link provided will almost certainly break in a few months)